verification/verify_setlist.py

```
@@ -0,0 +1,73 @@
+            print("Navigating to http://localhost:5173/...")
+            page.goto("http://localhost:5173/")
+
+            # Start Tour
+            page.wait_for_selector("text=Start Tour", timeout=10000)
+            page.click("text=Start Tour")
+
+            # Wait for Overworld
+            page.wait_for_selector("text=TOUR PLAN", timeout=15000)
+            print("In Overworld.")
+
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)